general-configuration-guide/basic-setup.md

```diff
@@ -84,12 +84,12 @@ tinymce.init({
 });
 ```
 
-We will show you how to configure plugins throughout the Get Started guide. The full list of plugins, their options and control associations is [available here]({{ site.baseurl }}/plugins/).
+We show you how to configure plugins throughout this guide. The full list of plugins, their options, and control associations is [available here]({{ site.baseurl }}/plugins/).
 
 
 ## Toolbar & menu configuration
 
-TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Basically, the type of settings you would expect to find in any WYSIWYG editor.
+TinyMCE comes with a default set of toolbar controls out of the box, such as bold, italic, and text alignment buttons. Ideally, the type of settings you would expect to find in any WYSIWYG editor.
```

Comment:
Do we even need the second sentence here?

Reply:
I think the second sentence makes a point that tells the user what to expect. I'm keeping it for now.